### DISCUSSION TITLE
WebView 2 on Xbox Message saying that WebView2 is not installed

### DISCUSSION THREAD

**Discussion by ShatsDev** [MicrosoftEdge/WebView2Feedback#4128] [Q&A] [answered]
Microsoft has announced WebView2 support on Xbox.
My app, [One Page Browser](https://apps.microsoft.com/detail/9MXS3495H1JG), is based on WebView2 but if I open it on Xbox I see the message talking that WebView2 is not installed.
What is wrong?

**Comment by victorhuangwq** [comment]
WebView2 is installed on Xbox October 2310 version (231018-2200). Could you be having an outdated version for Xbox?

**Reply by ShatsDev** [reply]
Looks updated.
OS version: 10.0.25398.2876 (xb_flt_2311zn.231102-1830)
2311.0.02311.2001

By the way, I cannot find the app in the Xbox store, but I can open Edge Microsoft store page, click Install button there and it will launch installation of the app. Weird.

**Comment by nishitha-burman** [comment]
Hi @AlexRUiLs, can you please follow the steps here and see if it works? https://github.com/MicrosoftEdge/WebView2Feedback/issues/2155#issuecomment-1789523587

**Reply by ShatsDev** [reply]
I am using Microsoft.UI.Xaml. Should I Install Microsoft.Web.WebView2 in addition to Microsoft.UI.Xaml?

**Reply by victorhuangwq** [reply]
did you give that a try? did it work?

**Reply by ShatsDev** [reply, marked as answer]
I updated both packages and it worked. Not sure if I have to add WebView2 package but I added.

**Comment by Sentry-Thibault** [comment]
Hi, 
Up on this issue. 

I am trying to launch my UWP app on my Xbox Dev Kit but a message tells me that "WebView 2 is not installed"
(Maybe Dev kit is a problem here)
Thanks

**Reply by victorhuangwq** [reply]
Did you make sure that your Xbox Dev Kit is up to date?

**Reply by victorhuangwq** [reply]
This is the version that is needed Xbox October 2310 version (231018-2200).

**Reply by Sentry-Thibault** [reply]
Good version installed. Dev Kit up to date too.
